Adding index.html w/ header and footer

ashwoodaa opened the pull request:
just for clarification: my branch name is "stopHeaderTime" like Stop! Hammer Time

ashwoodaa committed:
Adding index.html w/ header and footer

josephkane commented:
:+1:

MeganWord commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    </div>
+	</footer>
+</body>
+</html>
```
github recommends always having a last blank line at the end of your files- some older browsers and people using linux need it I believe :)

ashwoodaa replied:
Sounds good. I'll do that to all of the files. Thanks! :new_moon:

MeganCFord commented on index.html:
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
```
is your stuff using spaces not tabs, and using 2 spaces instead of 4? I'm using only spaces and 2 spaces for each nest, which means if you're using tabs or 4 spaces there could be compatibility issues later.

MeganWord replied:
is your stuff using spaces not tabs, and using 2 spaces instead of 4? I'm using only spaces and 2 spaces for each nest, which means if you're using tabs or 4 spaces there could be compatibility issues later.

ashwoodaa replied:
Nope. I have  "Indent using Spaces" and tab size: 2

MeganWord commented on index.html:
```diff
@@ -0,0 +1,40 @@
+		</a>
+		<nav>
+      <div id="topNav">
+  			<ul>
```
the 'li' list items need to be nested inside this UL.

ashwoodaa replied:
Already done. It's showing up weird on GitHub.

MeganWord commented on index.html:
```diff
@@ -0,0 +1,40 @@
+			<h3>Get your place in Outer Space!</h3>
+		</a>
+		<nav>
+      <div id="topNav">
```
note that the 'nav' semantic tag is technically a div- you might not need this one since Joe can select the nav bar using the nav tag. up to you guys though.

ashwoodaa replied:
Coolio. Joe wants it in the nav. So, it is done. Also done for the bottom nav  in the footer

MeganWord commented on index.html:
```diff
@@ -0,0 +1,40 @@
+		</nav>
+	</header><!-- /header -->
+	<footer>
+    <div id="bottomNav">
```
this div needs to be nested inside the footer div.

ashwoodaa replied:
Already done. It's showing up weird on GitHub.

ashwoodaa committed:
Added a final blank line and Moved id's to nav    moved the "topNav" and
"bottomNav" tags to the nav tag instead of the div tag in the header and
footer

MeganCFord commented:
:+1

MeganWord commented:
:+1:

ashwoodaa committed:
making sure that the id "topNav" in on the nav tag instead of the div tag

josephkane commented:
:+1:
